recipes/neuralprophet/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "neuralprophet" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/neuralprophet-{{ version }}.tar.gz
+  sha256: b6f0ecafa8c332c961f071a8664e1f6ba481822012b9cbd9352033cde9b90d4e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: win64
+  
+requirements:
+  host:
+    - python >=3.7,<3.11
+    - poetry-core
+    - pip
+  run:
+    - python >=3.7.1,<3.11
+    - captum >=0.6.0,<0.7.0
+    - holidays >=0.21.0,<0.22.0
+    - matplotlib-base >=3.5.3,<4.0.0
+    - numpy >=1.21.6,<1.24.0
+    - pandas >=1.3.5,<2.0.0
+    - plotly >=5.13.1,<6.0.0
+    - plotly-resampler >=0.8.3,<0.8.4
+    - pytorch-lightning >=1.9.4,<2.0.0
+    - tensorboard >=2.11.2,<3.0.0
+    - pytorch >=1.13.1,<2.0.0
+    - torchmetrics >=0.11.3,<0.12.0
+    - typing-extensions >=4.5.0,<5.0.0
+
+test:
+  imports:
+    - neuralprophet
+  commands:
+    # fails because of dash
+    # - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ourownstory/neural_prophet
+  summary: NeuralProphet is an easy to learn framework for interpretable time series forecasting.
+  license: MIT
+  license_file: LICENSE.txt
```

This is the name of the file in the tarball.

```suggestion
  license_file: LICENSE
```